docs/filters.md

```diff
@@ -31,19 +31,19 @@ will be one of:
 # parseFilter(filterString)
 
 Parses an [RFC2254](http://www.ietf.org/rfc/rfc2254.txt) filter string into an
-ldapjs object(s).  If the filter is "complex", it will be a "tree" of objects.
+ldapjs object(s). If the filter is "complex", it will be a "tree" of objects.
 For example:
 
 ```js
-const parseFilter = require('ldapjs').parseFilter;
+const parseFilter = require("ldapjs").parseFilter;
```

Comment:
Please turn off your autocorrect tool and revert the changes not relevant to the issue.

Reply:
ohh, i didn't notice that.
corrected it, can you check?